packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -153,7 +155,9 @@ export class JobService {
   ): Promise<CvatManifestDto> {
     const { getElementsCount } = this.createDatasetActions[requestType];
 
-    const elementsCount = await getElementsCount(requestType, dto.data);
+    const gtUrl = generateBucketUrl(dto.groundTruth, requestType);
+    const elementsCount = await getElementsCount(requestType, dto.data, gtUrl);
```

Comment:
elementsCount is obtained here only to be passed into calculateJobBounty, so I think we can get it inside calculateJobBounty